Mlem/Views/Tabs/Feeds/Community List/Components/CommunityListRowViews.swift

```diff
@@ -115,7 +113,9 @@ struct CommuntiyFeedRowView: View {
     }
 
     private func isFavorited() -> Bool {
-        favoriteCommunitiesTracker.favoriteCommunities(for: appState.currentActiveAccount).contains(community)
+        favoriteCommunitiesTracker.currentFavorites
```

Comment:
Add this logic in a `FavoriteCommunitiesTracker` extension?  Already duplicated in SidebarEntry, and probably will be used more in the future elsewhere.